src/services/behaviors/helpers.ts

```diff
@@ -0,0 +1,81 @@
+/**
+ * Copyright (c) Garuda Labs, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import {
+  ACTIONS,
+  ATTRIBUTES,
+  HvComponentOnUpdate,
+  NAV_ACTIONS,
+  NavAction,
+  UPDATE_ACTIONS,
+  UpdateAction,
+} from 'hyperview/src/types';
+
+export const triggerBehaviors = (
```

Comment:
As we're creating these new services, I'm curious what's the overlap between the new `triggerBehaviors` method and the `trigger` method available in the service's index. Maybe more just an naming issue that creates ambiguity

Reply:
There is decent overlap, but there are differences in their usage. We may want to create a separate task to consolidate these.